client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/interfaces.ts

```diff
@@ -32,3 +32,7 @@ export interface SitesViewState {
 	layout: object;
 	selectedSite?: Site | undefined;
 }
+
+export interface SiteInfo extends SiteData {
+	id: number;
```

Comment:
We need the site items with an accessible `id`. We have to do it because of a bug in the DataViews component for the bulk selection system, that it tries to get the `id` instead of using the `getItemId` method.